.travis.yml

```diff
@@ -1,18 +1,26 @@
-language: python
+language: c
 
-python:
-    - 2.7
+os:
+    - osx
+    - linux
 
-env: DISPLAY=:99 MOZ_XVFB=1
+compiler:
+    - gcc
```

Comment:
I see. Lets make a comment out of this information, so that it is clear at this point. Thanks for that info!